src/Chapter2.hs

```diff
@@ -492,50 +395,42 @@ the same as writing "(x:(y:xs))".
 
 {- |
 =⚔️= Task 5
-
 Implement a function that checks whether the third element of a list
 is the number 42.
-
 >>> isThird42 [1, 2, 42, 10]
 True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: [a] -> Bool
+isThird42 (_:_:42:_) = True
```

Comment:
I see that this code doesn't compile. In the type signature, you write that the function `isThird42` works with any type `a`, but in the implementation, in pattern, you match on number 42. Not every type is a number. To fix this problem, specify the type of a function to accept a list of integers:

```haskell
isThird42 :: [Int] -> Bool
```